tiledb/bioimg/converters/base.py

```diff
@@ -183,12 +184,9 @@ def from_tiledb(
 
         with tiledb.scope_ctx(config):
             vfs_use = output_path.startswith("s3://")
-            if vfs_use:
-                vfs = tiledb.VFS()
-                destination_uri = vfs.open(output_path, "wb")
-            else:
-                destination_uri = output_path
-
+            destination_uri = (
```

Comment:
Also for the `vfs_use`, lets just use vfs for all cases now. Or we need to use it for azure, gcs, hdfs, s3 at a minimum.

Reply:
We need to do the latter. The reason for `vfs use` flag is just to exclude local files (`scrach_space`) from be handled by `vfs` which in `tifffile` activates a different write path that uses the VFS IO (`write` especially). So I guess I can inverse the check so all the remote `fs` can be handled the same as `s3` now.